openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -37,6 +36,7 @@ def get_api_client(api_config, student):
     return EdxRestApiClient(api_config.internal_api_url, jwt=id_token)
 
 
+# TODO: Move to program utilities. Will require updating tests.
```

Comment:
Fair point. I feel strongly for not leaving the code in this state, but I agree a separate PR would be cleaner and easier to gain value earlier. Maybe we should not consider the ticket is done until the new refactor PR is merged?